examples/with-multi-content-types/deletePost.go

```diff
@@ -25,19 +25,18 @@ func DeletePostJSON(c *gin.Context) {
 		return
 	}
 
-	c.JSON(http.StatusOK, deleteOperation(postURI))
-}
-
-func DeletePostYAML(c *gin.Context) {
-	var postURI types.PostID
-	err := c.ShouldBindUri(postURI)
-	if err != nil {
-		c.JSON(http.StatusBadRequest, types.Error{
-			Error:   "cannot bind uri",
-			Details: err.Error(),
-		})
-		return
+	switch c.ContentType() {
+	case "application/json":
+		{
```

Comment:
These work well in JS I get it, but personally I don't like them here. Go FMT does us no favours and frankly I'm not a fan.

Reply:
Hmmm, I quite liked them, I know there quiet large but im not sure that's a problem here?

Reply:
I have removed the surrounding curly braces on the cases which I think has improved the look of the switch?

Reply:
Cool, think it looks fine now.